docs/frontend-templates/document-collections.md

```diff
@@ -41,6 +41,13 @@ howItWorks:
   - publications in the same series
 
   - a mix of document types related to the same task or event
+
+  ### Contents list
+
+  The contents list on a document collection page is created automatically from the groups of documents on the page. Each group has a H2 that appears in the contents list.
```

Comment:
If you want to use single quote will need to use `‘` as VS Code doesn't recognize the single quote in a "literal"  written sense.